### ISSUE TITLE
Recalls and alerts: The "alt " attribute is not empty for icon

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#165]
**Page:** Find recalls, advisories and safety alerts
**URL:** https://rsams.dev.cloud.openplus.ca/en
**User Agent:** Firefox (v99)
**Language:** English and French

**Issue Description:** The icon associated with an alert or recall doesn't provide an empty "alt" attribute. In this case, the same or similar information about alert or recall is provided below it and the icon is considered as a decorative image.

Further, on the French pages, the "alt" attribute value is in English for the icons.

![image](https://user-images.githubusercontent.com/12256067/162517969-942726b7-924d-4047-8d13-e7b3ca9bdf1d.png)

![image](https://user-images.githubusercontent.com/12256067/162518408-fa3deb48-aba8-4b74-b4bf-8ffb8b0a395c.png)

**Steps to reproduce:**

1. Go to the above URL
2. Visual scanning and inspect the HTML source

**Expected results:** Since the similar information is provided visually, the icons should have empty "alt" attributes.

**Comment by YuxuanXie01** [comment]
Retested, partial fix. Right now each icon doesn't have a description but icons are mistakenly tagged with alt="" "" on both the English and French pages, making screen readers to incorrectly announce it as 'quote space quote.' This attribute should be corrected to alt="" to ensure that screen readers correctly ignore these icons.
![image](https://github.com/user-attachments/assets/95d95e59-e4ff-4ffb-9d8c-662cafa07d7a)
![image](https://github.com/user-attachments/assets/d25dec6f-ad52-4eb7-980b-55015ca2bff5)

**Comment by zimw7** [comment]
The alt text is still incorrect. It shows as alt="" "" in both English and French versions, with extra double quotes instead of alt=" "
![image](https://github.com/user-attachments/assets/cdef8270-7789-476f-912b-c4743f4b37d3)
![image](https://github.com/user-attachments/assets/d729d57d-2a5b-4177-9db1-f2b467a1a819)

**Comment by 40999221** [comment]
Partially fixed : alternate text were removed but replaced by double quotes which catches in screen reader ( "quote space quote"). Instead of this, they need to remove the quotes by alt="" .
![image](https://github.com/user-attachments/assets/74ba0bef-f2ec-488a-a468-3932b4a50860)

**Comment by mmiron-HC** [comment]
@40999221 Please copy this ticket over to [ART-2605](https://jill.hc-sc.gc.ca/jira/browse/ART-2605)
